### DISCUSSION TITLE
bagaimana cara menambah simbol $ dan format ribuan pada kolom price_increase?

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#702] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/23212456/ce718714-a4ff-49a4-9395-b37263f8070b)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @Gustana 

Untuk menambahkan $ dan koma pada angka numerik dapat menggunakan fungsi `dollar()` dari library `scales`, sebagai contoh seperti berikut.

Di sini kita mempunyai `df` yang memiliki kolom `current_number`
![image](https://github.com/teamalgoritma/community/assets/56690035/6feb3e37-c01d-4a0b-b115-2ffde6e4caaf)

Nah setelah itu kita menggunakan fungsi `scales::dollar()` dengan memasukkan nama kolom numerik yang ingin diubah formatnya seperti berikut.
![image](https://github.com/teamalgoritma/community/assets/56690035/e4555994-964f-4dac-96d5-2300e90d6a8a)

Fungsi `scales::dollar()` sendiri memiliki banyak paramter yang bisa diubah, termasuk `prefix` untuk mengganti `$`, mngkn seperti `Rp.` atau pun mata uang lainnya. Dokumentasi dapat mengacu ke section "Help" pada R Studio 

Sekian jawaban dari kami, semoga membantu.
